openstack/telemetry/v2/meters/requests.go

```diff
@@ -0,0 +1,92 @@
+package meters
+
+import (
+	"github.com/rackspace/gophercloud"
+)
+
+// ListOptsBuilder allows extensions to add additional parameters to the
+// List request.
+type ListOptsBuilder interface {
+	ToMeterListQuery() (string, error)
+}
+
+// ListOpts allows the filtering and sorting of collections through
+// the API. Filtering is achieved by passing in struct field values that map to
+// the server attributes you want to see returned.
+type ListOpts struct {
+}
+
+// ToServerListQuery formats a ListOpts into a query string.
+func (opts ListOpts) ToMeterListQuery() (string, error) {
+	q, err := gophercloud.BuildQueryString(opts)
+	if err != nil {
+		return "", err
+	}
+	return q.String(), nil
+}
+
+// List makes a request against the API to list meters accessible to you.
+func List(client *gophercloud.ServiceClient, opts ListOptsBuilder) listResult {
```

Comment:
It would be a page, but the API response body does not provide a link to next page, therefore it would be impossible to code ```NextPageURL()``` function and in my understanding, paging wouldn't work automagically on gophercloud's side. How should I solve that?